documentation/developer-guide/modules/tooling-guide/pages/bamm-cli.adoc

```diff
@@ -3,81 +3,88 @@
+| aspect <model> to html                | Generate HTML documentation for an Aspect Model                | bamm-cli aspect AspectModel.ttl to html
+|                                       |    _--output, -o_ : the output will be saved to the given file | bamm-cli aspect AspectModel.ttl to html -o c:\Model.html
+|                                       |    _--css, -c_ : CSS file with custom styles to be included in the generated HTML documentation | bamm-cli aspect AspectModel.ttl to html -c c:\styles.css
+|                                       |    _--language, -l_ : The language from the model for which the html should be generated (default: en) | bamm-cli aspect AspectModel.ttl to html -l de
+| aspect <model> to png                 | Generate PNG diagram for Aspect Model                          | bamm-cli aspect AspectModel.ttl to png
+|                                       |    _--output, -o_ : output file path (default: stdout); as PNG is a binary format, it is strongly recommended to output the result to a file by using the -o option or the console redirection operator '>')|
+|                                       |    _--language, -l_ : the language from the model for which the diagram should be generated (default: en)|
+| aspect <model> to svg                 | Generate SVG diagram for Aspect Model                          | bamm-cli aspect AspectModel.ttl to svg
+|                                       |    _--output, -o_ : the output will be saved to the given file |
+|                                       |    _--language, -l_ : the language from the model for which the diagram should be generated (default: en)|
+| aspect <model> to dot                 | Generate DOT diagram for Aspect Model                          | bamm-cli aspect AspectModel.ttl to dot
+|                                       |    _--output, -o_ : output file path (default: stdout)         |
+|                                       |    _--language, -l_ : the language from the model for which the diagram should be generated (default: en)|
+| aspect <model> to java                | Generate Java classes from an Aspect Model                     | bamm-cli aspect AspectModel.ttl to java
+|                                       |  _--output-directory, -d_ : output directory to write files to (default: current directory)|
+|                                       |  _--package-name, -pn_ : package to use for generated Java classes| bamm-cli aspect AspectModel.ttl to java -pn org.company.product
+|                                       |  _--no-jackson, -nj_ : disable Jackson annotation generation in generated Java classes|
+|                                       |  _--template-library-file, -tlf_ : the path and name of the velocity template file containing the macro library|
```

Comment:
"Velocity" (in upper case), also turn Velocity into hyperlink to https://velocity.apache.org/